lib/src/components/NxFileUpload/NxFileUpload.scss

```diff
@@ -47,6 +47,34 @@
   }
 }
 
+
+.nx-multi-file-upload {
+  width: 348px;
+
+  // need an enclosing div around NxMultiFileUpload that doesn't contain the validationError message
+  &__container {
+    $height: 220px;
+
+    min-height: $height;
+    box-sizing:border-box;
+    border: 1px solid var(--nx-swatch-indigo-90);
+    border-radius: 6px;
+
+    &__files {
+      padding: var(--nx-spacing-6x);
+      padding-bottom: 0px;
+      // total height of NxMultiFileUpload - total height of button with margin
+      max-height: calc(#{$height} - 90px);
```

Comment:
> Something that can be fixed as well, but a heads up that by adding display:flex to nx-multi-file-upload, it affects the positioning of the validation message, so it would need to be adjusted for nx-multi-file-upload as well.

I meant like this: https://github.com/sonatype/sonatype-react-shared-components/commit/32a3d52230db32f6fa9272c610e8591037c3c4d3

The max-height that `.nx-scrollable` has by default is larger than the height that the scrollable will have here, so it doesn't really come into play.

Reply:
> I meant like this: [32a3d52](https://github.com/sonatype/sonatype-react-shared-components/commit/32a3d52230db32f6fa9272c610e8591037c3c4d3)
> 
> The max-height that `.nx-scrollable` has by default is larger than the height that the scrollable will have here, so it doesn't really come into play.

Thanks for the tip, implemented here: [259bdb2](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/259bdb22c3ddc9356858bbec1ccefd6e66f7265c)